src/lib/skeleton/skeleton.ts

```diff
@@ -5,7 +5,38 @@ import { SKELETON_CONSTANTS } from './skeleton-constants';
 import template from './skeleton.html';
 import styles from './skeleton.scss';
 
-export interface ISkeletonComponent extends IBaseComponent {}
+/**
+ * @summary Skeleton is used to provide a placeholder for content that is loading.

```

Comment:
This needs to be on the `class` below. You can merge it with that, but remove the `"The web component class behind the `<forge-skeleton>` custom element."` text when you do.